lenses/sudoers.aug

```diff
@@ -88,7 +88,7 @@ let sep_dquote   = Util.del_str "\""
 (* Variable: sto_to_com_cmnd
 sto_to_com_cmnd does not begin or end with a space *)
 let sto_to_com_cmnd =
-      let alias = Rx.word - /(NO)?(PASSWD|EXEC|SETENV)/
+      let alias = /!?/ . Rx.word - /(NO)?(PASSWD|EXEC|SETENV)/
```

Comment:
I'd rather use the `negate` and `negate_node` lenses, so it behaves like the rest of the lens:

```augeas
let alias = del_negate . negate_node? . Rx.word - /(NO)?(PASSWD|EXEC|SETENV)/
```